src/index.ts

```diff
@@ -135,29 +135,29 @@ client.on('ready', async () => {
     loadCmds();
     function loadMods(): void {
```

Comment:
```suggestion
    function loadHooks(): void {
```

Also change line 166 manually (I can't)